src/main/java/com/based/basedsurvey/controller/WebEditSurveyController.java

```diff
@@ -55,6 +55,16 @@ public String deleteQuestion(@RequestParam long questionID, @RequestParam long s
         return "redirect:/survey/"+surveyID +"/edit";
     }
 
+    @DeleteMapping(value = "question/delete/{questionID}")
+    @ResponseBody
+    public String deleteSurveyHtmx(@PathVariable long questionID){
```

Comment:
What exactly does the @responsebody string do given that you're returning a string

Reply:
it makes it so we don't return the path to a html view.